qiita_pet/handlers/study_handlers/tests/test_base.py

```diff
@@ -59,5 +59,37 @@ def test_get(self):
         self.assertNotEqual(response.body, "")
 
 
+class TestStudyGetTags(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/study/get_tags/')
+        exp = ('{"status": "success", "message": "", "tags": '
+               '{"admin": [], "user": []}}')
+        self.assertEqual(response.code, 200)
+        self.assertEqual(response.body, exp)
+
+
+class TestStudyTags(OauthTestingBase):
+    def test_get(self):
+        response = self.get('/study/tags/1')
+        exp = ('{"status": "success", "message": "", "tags": []}')
+        self.assertEqual(response.code, 200)
+        self.assertEqual(response.body, exp)
+
+    def test_patch(self):
```

Comment:
I don't think is required as all possible options are tested here: https://github.com/biocore/qiita/pull/2088/files#diff-a0b367a8e2e2e1d72fee16533d797415R541, what do you think?